content/post/career_ladders.md

```diff
@@ -0,0 +1,97 @@
+---
+author: Caitlin Moorman
+title: "Career Ladders in Analytics"
+cover: ""
+draft: true
+publishDate: "2019-12-15"
+tags:
+ - analytics
+ - org
+ - teams
+ - management
+---
+
+When I started my career, I didn’t have a career ladder, and I didn’t exactly understand the point of them. I naively assumed that if I did good work, I would be acknowledged for it. I wasn’t always sure what good work meant, but I had enough supportive colleagues around me that they usually let me know when I was heading in the wrong direction. It mostly worked - I got promoted a few times (each time a surprise to me), shifted to an entirely new field, and got promoted a few more times. Honestly, most of those promotions came as a pleasant surprise - while I love learning new things and generally tried to work on whatever was most useful to the company, I didn’t have much guidance on what the “next level” looked like.
+
+I didn’t truly appreciate the value of a career ladder until I started to lead a team. The perspective I have now, having helped each person on my team use a career ladder to find development opportunities and work toward promotions, would have been really handy when I stumbled through the first decade of my career. To that end, this post will outline what a career ladder is, why you need one, my framework for thinking about career levels for analysts, and what I should have done 10 years ago to get more clear direction on what’s next as an individual contributor without a career ladder.
+
+## What Is a Career Ladder?
+A career ladder is a framework to document the path or paths an individual can take as they develop within their company. It describes different roles (hopefully both individual contributor roles and manager roles), outlines the expectations for each, and provides sufficiently detailed information to see what is needed to move from one role to another. A career ladder can include hard skills, as well as an outline of *how* one should work in each role and the type of impact that role should have.
+
+## Why Clear Career Paths Matter
+Analysts, of all people, want to know how they are being measured and how to reach the next level. Clearly outlining the path ahead is critical to keeping strong performers. Sometimes, it is explicitly about how to get the next promotion, but it’s also about how to increase impact, measure growth, and think about future options.
```

Comment:
Nit: 

>is critical to keeping strong performers.

Maybe sub "keeping" for "retaining" or extend the sentence to something like

> is critical to keeping strong performers engaged and progressing